python/ray/_private/services.py

```diff
@@ -492,18 +492,16 @@ def get_node(gcs_address, node_id):
     return global_state.get_node(node_id)
 
 
-def get_webui_url_from_internal_kv():
-    assert ray.experimental.internal_kv._internal_kv_initialized()
-    webui_url = ray.experimental.internal_kv._internal_kv_get(
-        "webui:url", namespace=ray_constants.KV_NAMESPACE_DASHBOARD
+def get_webui_url_from_internal_kv(gcs_client: GcsClient):
+    webui_url = gcs_client.internal_kv_get(
```

Comment:
A small cleanup to avoid depending on `ray.experimental.internal_kv` but `GcsClient` directly so we don't need to rely on `internal_kv` being initialized.